packages/aft/test/version_bump/data/snapshots/single_dart_package_feat.diff

```diff
@@ -1,6 +1,3 @@
-diff --git a/packages/auth/amplify_auth_cognito/pubspec.yaml b/packages/auth/amplify_auth_cognito/pubspec.yaml
--  amplify_auth_cognito_dart: ">=0.11.3 <0.12.0"
-+  amplify_auth_cognito_dart: ">=0.11.4 <0.12.0"
```

Comment:
Note to reviewers: This is another example of a bug being fixed where a dependency constraint is made when it does not need to. The existing version constraint (`">=0.11.3 <0.12.0"`) is valid. Prior to this PR, this diff would have been problematic since amplify_auth_cognito has a constraint update, but is not published.